client/music.cpp

```diff
@@ -62,6 +63,7 @@ void start_style_music()
       if (!tag.isEmpty()) {
         log_debug("Play %s", qUtf8Printable(tag));
         audio_play_music(tag, nullptr, MU_INGAME);
+        show_music_info(tag);
```

Comment:
What's the content of `tag`? Do we want some more text, like "Now playing..."?

Reply:
Ya, we do. Tag is just a unique identifier currently. If it works, I'll tweak it to show something nice.